pkg/interface/src/views/apps/publish/components/MetadataForm.tsx

```diff
@@ -64,11 +64,11 @@ export function MetadataForm(props: MetadataFormProps) {
       const { name, description } = values;
       await api.metadata.metadataAdd(
         "publish",
-        props.association["app-path"],
-        props.association["group-path"],
+        props.association.resource,
+        props.association.group,
         name,
         description,
-        props.association.metadata["date-created"],
+        props.association.metadata["date-created"],,
```

Comment:
Does this compile?

Reply:
Hahah, just realised this component is no longer used. Removing.